packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1010,14 +1010,13 @@ describe('plugin-meetings', () => {
             state: 'JOINED',
           };
           meeting.areVoiceaEventsSetup = false;
-          meeting.roles = ['COHOST'];
 
           await meeting.startTranscription();
 
           assert.equal(webex.internal.voicea.on.callCount, 4);
           assert.equal(meeting.areVoiceaEventsSetup, true);
           assert.equal(webex.internal.voicea.listenToEvents.callCount, 1);
-          assert.notCalled(webex.internal.voicea.turnOnCaptions);
+          assert.calledOnce(webex.internal.voicea.turnOnCaptions);
```

Comment:
Can we update the test description as well?

Reply:
Thanks, @rarajes2. Good catch. Updated the test description